unittests/test_kernels.py

```diff
@@ -1,1009 +1,968 @@
-    furnsh(PATH_ + 'sat164.bsp')
-    furnsh(PATH_ + 'earth_000101_180317_171224.bpc')
-
-    furnsh(PATH_ + 'cas_v40.tf')
-    furnsh(PATH_ + 'cas00171.tsc')
-    furnsh(PATH_ + '17257_17262ra.bc')
-    furnsh(PATH_ + '171106R_SCPSE_17224_17258.bsp')
-    furnsh(PATH_ + 'cas_iss_v10.ti')
-
-  @staticmethod
-  def unload_kernels():
-    unload(PATH_ + 'naif0012.tls')
-    unload(PATH_ + 'pck00010.tpc')
-    unload(PATH_ + 'de432s.bsp')
-    unload(PATH_ + 'sat164.bsp')
-    unload(PATH_ + 'earth_000101_180317_171224.bpc')
-
-    unload(PATH_ + 'cas_v40.tf')
-    unload(PATH_ + 'cas00171.tsc')
-    unload(PATH_ + '17257_17262ra.bc')
-    unload(PATH_ + '171106R_SCPSE_17224_17258.bsp')
-    unload(PATH_ + 'cas_iss_v10.ti')
-
-  def assertAllEqual(self, arg1, arg2, tol=1.e-15, frac=False):
-    if isinstance(arg1, list):
-        self.assertEqual(type(arg2), list)
-        self.assertEqual(len(arg1), len(arg2))
-        for (item1,item2) in zip(arg1,arg2):
-            self.assertAllEqual(item1, item2, tol, frac)
-
-    elif isinstance(arg1, np.ndarray):
-        arg1 = np.array(arg1)
-        arg2 = np.array(arg2)
-        self.assertEqual(arg1.shape, arg2.shape)
-        arg1 = arg1.flatten()
-        arg2 = arg2.flatten()
-        for (x1,x2) in zip(arg1,arg2):
-            if isinstance(x1,numbers.Real):
-                if frac:
-                    self.assertTrue(abs(x1 - x2) <= tol * abs(x1 + x2)/2.)
+import pytest
+
+PATH_ = Path(os.path.realpath(__file__)).parent.parent / "unittest_support/"
+
+def furnish_kernels():
+    furnsh(PATH_ / 'naif0012.tls')
+    furnsh(PATH_ / 'pck00010.tpc')
+    furnsh(PATH_ / 'de432s.bsp')
+    furnsh(PATH_ / 'sat164.bsp')
+    furnsh(PATH_ / 'earth_000101_180317_171224.bpc')
+
+    furnsh(PATH_ / 'cas_v40.tf')
+    furnsh(PATH_ / 'cas00171.tsc')
+    furnsh(PATH_ / '17257_17262ra.bc')
+    furnsh(PATH_ / '171106R_SCPSE_17224_17258.bsp')
+    furnsh(PATH_ / 'cas_iss_v10.ti')
+
+
+def unload_kernels():
+    unload(PATH_ / 'naif0012.tls')
+    unload(PATH_ / 'pck00010.tpc')
+    unload(PATH_ / 'de432s.bsp')
+    unload(PATH_ / 'sat164.bsp')
+    unload(PATH_ / 'earth_000101_180317_171224.bpc')
+
+    unload(PATH_ / 'cas_v40.tf')
+    unload(PATH_ / 'cas00171.tsc')
+    unload(PATH_ / '17257_17262ra.bc')
+    unload(PATH_ / '171106R_SCPSE_17224_17258.bsp')
+    unload(PATH_ / 'cas_iss_v10.ti')
+
+
+@pytest.fixture(autouse=True)
+def kernels():
+    furnish_kernels()
+    yield
+    unload_kernels()
+
+@pytest.fixture
+def CASSINI_ET():
+    return 17.65 * 365.25 * 86400.
+
+@pytest.fixture
+def CASSINI_ET2(CASSINI_ET):
+    return CASSINI_ET + 86400
+
+@pytest.fixture
+def eps():
+    if platform.machine() == 'arm64':
+        # TODO(fy,rf,mrs): This code gives slightly different results on MacOS Arm64.
+        # This lowering of expectations needs to be investigated. Is it Mac only?
+        eps = 1e-5
+    else:
+        eps = 5e-8
+    return eps
+
+
+class Test_cspyce1_kernels:
```

Comment:
Is there still a reason to have these as methods in a class? Not wrong, just wondering.